libs/dataset-explorer/src/lib/FeatureBalanceMeasuresChart.tsx

```diff
@@ -0,0 +1,232 @@
+import {
+  HeatmapHighChart,
+  IFeatureBalanceMeasures,
+  LabelWithCallout
+} from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
+import _ from "lodash";
+import React from "react";
+
+import { dataBalanceTabStyles } from "./DataBalanceTab.styles";
+import {
+  FeatureBalanceMeasuresMap,
+  getFeatureBalanceMeasuresChart
+} from "./getFeatureBalanceMeasuresChart";
+
+export interface IFeatureBalanceMeasuresProps {
+  featureBalanceMeasures: IFeatureBalanceMeasures;
+}
+
+export interface IFeatureBalanceMeasuresState {
+  selectedFeatureIndex: number;
+  selectedMeasureIndex: number;
+}
+
+export class FeatureBalanceMeasuresChart extends React.PureComponent<
+  IFeatureBalanceMeasuresProps,
+  IFeatureBalanceMeasuresState
+> {
+  public constructor(props: IFeatureBalanceMeasuresProps) {
+    super(props);
+
+    this.state = {
+      selectedFeatureIndex: 0,
+      selectedMeasureIndex: 0
+    };
+  }
+
+  public render(): React.ReactNode {
+    const featureBalanceMeasures = this.props.featureBalanceMeasures;
+    if (!featureBalanceMeasures) {
+      return;
+    }
+
+    const styles = dataBalanceTabStyles();
+    const measuresLocalization =
+      localization.ModelAssessment.DataBalance.FeatureBalanceMeasures;
+
+    const featureOptions = _.uniq(Object.keys(featureBalanceMeasures)).map(
+      (feature, index) => ({ key: index, text: feature } as IDropdownOption)
+    );
+    const selectedFeature =
+      featureOptions[this.state.selectedFeatureIndex].text;
+
+    const measureOptions = [...FeatureBalanceMeasuresMap.keys()].map(
+      (measure, index) => ({ key: index, text: measure } as IDropdownOption)
+    );
+    const selectedMeasure =
+      measureOptions[this.state.selectedMeasureIndex].text;
+
+    return (
+      <Stack tokens={{ childrenGap: "l1" }}>
+        {/* Renders the title and info hover-over */}
+        <Stack.Item>
+          <Stack horizontal>
+            <Stack.Item>
+              <Text variant="large" className={styles.boldText}>
+                {measuresLocalization.Name}
+              </Text>
+            </Stack.Item>
+
+            <Stack.Item className={styles.callout}>
+              <LabelWithCallout
+                label=""
+                calloutTitle={measuresLocalization.Callout.Title}
+                type="button"
+              >
+                <Text block>{measuresLocalization.Callout.Description}</Text>
+                <Link
+                  // TODO: Replace link with https://responsibleaitoolbox.ai/ link once docs are published there
+                  href="https://microsoft.github.io/SynapseML/docs/features/responsible_ai/Data%20Balance%20Analysis/#feature-balance-measures"
+                  target="_blank"
+                >
+                  {localization.ModelAssessment.DataBalance.LearnMore}
+                </Link>
+              </LabelWithCallout>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
```

Comment:
It feels like I just read this in the previous file. Can we refactor this into its own React component and have the specific parts be props?

Reply:
Great suggestion, done! :)